lxqt-base/liblxqt/liblxqt-0.13.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-utils eapi7-ver
+
+DESCRIPTION="Common base library for the LXQt desktop environment"
+HOMEPAGE="https://lxqt.org/"
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lxqt/${PN}.git"
+else
+	SRC_URI="https://downloads.lxqt.org/downloads/${PN}/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+fi
+
+LICENSE="LGPL-2.1+ BSD"
+SLOT="0/$(ver_cut 1-2)"
+
+RDEPEND="
+	dev-libs/libqtxdg:0/3
```

Comment:
sure you want to depend on the subslot there rather than trigger rebuild on subslot change?

Reply:
libqtxdg-2.0.0 is just slot 0, but liblxqt depends on version 3, so I was making sure there was no way for version 2 to be pulled in.

Reply:
Not sure what to do about this. Any suggestions? Thanks.

Reply:
Take the subslot out of libqtxdg-3.2.0 and just depend on >=dev-libs/libqtxdg-3.0 in liblxqt-0.13.0.
That way also the live-ebuild in the qt-overlay (libqtxdg-9999) could fulfill the dep requirement.

Reply:
i don't think the live ebuild in the overlay is a concern right now, that should get updated anyway once this PR is dealt with

Reply:
I changed it all to >=dev-libs/libqtxdg-3.0.0. I hope this is good.